deepchecks/nlp/input_validations.py

```diff
@@ -236,3 +240,209 @@ def compare_dataframes(
         difference = None
 
     return DataframesComparison(common, difference)
+
+
+def _validate_text_classification(
```

Comment:
_validate_text_classification_predictions
so its clear it does not validate anything else

Reply:
it validates not only predictions